pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/ParticipantDao.java

```diff
@@ -137,10 +141,16 @@ public void accumulate(Map<Long, Participant> container, RowView row) {
                 }
                 return new Participant(status, user);
             });
-
-            if (row.getColumn("m_user_medical_provider_id", Long.class) != null) {
+            Long medicalProviderId = row.getColumn("m_user_medical_provider_id", Long.class);
+            if (medicalProviderId != null
+                    && participant.getProviders().stream().noneMatch(p -> p.getUserMedicalProviderId() == medicalProviderId)) {
```

Comment:
I see here in the reducer that we're dealing with potential duplicate results. But maybe it's cleaner to have a separate query?